packages/react/use-loading/src/useLoading.spec.ts

```diff
@@ -80,7 +80,7 @@ describe('useLoading', () => {
     });
   });
   describe('(arg1: Arg1, arg2: Arg2) => Promise<Data> (type inference 테스트)', () => {
```

Comment:
Is there a reason why you left only this part in Korean?!

```suggestion
  describe('(arg1: Arg1, arg2: Arg2) => Promise<Data> (type inference test)', () => {
```

Reply:
No,,, It is my mistake,,,😇 thanks!